lib/server.js

```diff
@@ -101,8 +104,17 @@ exports.create = function create(createOptions) {
         });
       }
       else {
+        // normalzie absolute view file path and absolute base directory path
+        // to prevent path strings like `/folder1/folder2/../../folder3/exampleFile`
+        thing = path.normalize(thing);
+        options.settings.views = path.normalize(options.settings.views);
```

Comment:
instead of doing this for every render call, can we do this once during the initialization? for example here: https://github.com/paypal/react-engine/blob/master/lib/server.js#L46